docs/troubleshooting/semgrep-app.md

```diff
@@ -173,18 +173,16 @@ Check the log output for any hints about what the issue is.
   ![Screenshot of GitHub's Actions tab with workflows disabled](/img/github-workflows-disabled.png)
   - Enable workflows by clicking **I understand my workflows, go ahead and enable them** to allow Semgrep to scan.
 
-### If a project reports the last scan 'Never finished'
+### If a project reports a scan 'Never finished'
 
-This status means that your CI jobs start and authenticate correctly, but fail before completion.
+Most often, this status means that your CI jobs start and authenticate correctly, but fail or are canceled before completion. Check your CI provider (such as GitHub Actions) for the log output of the latest Semgrep job execution. In most cases you will see an error message with detailed instructions on what to do.
 
-Check your CI provider (such as GitHub Actions) for the log output of the latest Semgrep job execution. In most cases you will see an error message with detailed instructions on what to do.
+In some cases, it may be shown when the scan has been running for a long time (more than an hour) and is still in progress. Scans that eventually produce results will always be accepted by Semgrep AppSec Platform even if this message is shown.
```

Comment:
```suggestion
Sometimes, this status may be shown when the scan has been running for a long time (more than an hour) and is still in progress. Scans that eventually produce results will always be accepted by Semgrep AppSec Platform, even if this message is shown.
```

Minor wording change so it doesn't sound as redundant (given the last sentence of the previous paragraph)